docs/source/cupy-reference/difference.rst

```diff
@@ -0,0 +1,55 @@
+Difference between CuPy and NumPy
+=================================
+
+The interface of CuPy is designed to obey the interface of NumPy.
+However it has some differenece.
+
+
+Cast behavior from float to integer
+-----------------------------------
+
+Some casting behavior from float to integer is not defined.
+For example from a negative float to unsigned integer and infinity to integer.
+It depends on your CPU.
```

Comment:
How about rewrite it like this?

For example, the casting from negative float value to unsigned integer and the infinity to signed integer can be different in some environment. This is the result of Intel CPU:
...(example)...